data/scenario-01.json

```diff
@@ -1,5 +1,4 @@
 {
```

Comment:
We should write a test which parses these json blocks to check they're valid json

Reply:
https://github.com/bbc/simorgh/issues/89

Reply:
Agree with Benjamin